.github/workflows/push.yml

```
@@ -0,0 +1,141 @@
+          lfs: true
+      - name: Clone submodules
+        run: git submodule update --init --recursive
+      - name: Install g++-8
+        run: sudo apt-get install gcc-8 g++-8
+      - name: Run cmake on fuzzypp
+        run: cmake -DCMAKE_CXX_COMPILER=/usr/bin/g++-8 ./fuzzypp
+      - name: Run cmake build
+        run: cmake --build . -- -j4
+      - name: Run fuzzyppcli-tests
+        run: "./fuzzypp/bin/fuzzyppcli-tests"
+      - name: Zip artifact
+        run: zip -j ./fuzzyppcli.zip ./fuzzypp/bin/fuzzyppcli
+      - name: Upload zip artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: unix-zip
+          path: ./fuzzyppcli.zip
+
+  test-fuzzypp-windows:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - name: Clone submodules
+        run: git submodule update --init --recursive
+      - name: Run cmake on fuzzypp
+        run: cmake -G "Visual Studio 16 2019" -A "x64" ./fuzzypp
+      - name: Run cmake build
+        run: cmake --build . --config Release
+      - name: Run fuzzyppcli-tests
+        run: "./fuzzypp/bin/Release/fuzzyppcli-tests.exe"
+      - name: Zip artifact
+        run: 7z a -r -tzip ./fuzzyppcli-win.zip ./fuzzypp/bin/Release/fuzzyppcli.exe
+      - name: Upload zip artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: windows-zip
+          path: ./fuzzyppcli-win.zip
+
+  release:
+    needs: [test-codepropertygraph, test-fuzzypp-linux, test-fuzzypp-windows]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - name: Clone submodules
+        run: git submodule update --init --recursive
+      - name: Set up JDK 1.8
+        run: sudo apt-get install openjdk-8-jdk
+      - name: Install specific sbt version
+        run: sudo apt-get install -y --allow-downgrades sbt=1.3.12
+      - name: Print sbt version
+        run: sbt --version
+      - name: Run tests
+        run: sbt scalafmtCheck test
+      - name: Export git tag to ENVVAR
+        run: export LATEST_TAG=$(git describe --tags --abbrev=0)
+      - name: Echo git tag
+        run: echo $LATEST_TAG
+      - name: Tag release
+        run: sbt ciReleaseTagNextVersion
+      - name: Decrypt private-key.pem
+        run: openssl aes-256-cbc -K ${{ env.KEY_663dec309dc8 }} -iv ${{ env.IV_663dec309dc8 }} -in private-key.pem.enc -out private-key.pem -d
+      - name: Import GPG key
+        run: gpg --import --batch private-key.pem
+      - name: Publish `codepropertygraph` release to Sonatype
+        run: sbt ciReleaseSonatype
+      - name: Publish `codepropertygraph` release to ShiftLeft artifactory
+        run: sbt 'set publishTo := Some("releases" at "https://shiftleft.jfrog.io/shiftleft/libs-release-local")' "set credentials += Credentials(\"Artifactory Realm\", \"shiftleft.jfrog.io\", \"${{ secrets.JFROG_USER }}\", \"${{ secrets.JFROG_PASS }}\")" ciRelease
```

```suggestion
      - run: sbt test ciReleaseTagNextVersion ciReleaseSonatype
        env:
          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
          PGP_SECRET: ${{ secrets.PGP_SECRET }}
          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Those secrets are configured for the entire organization, so :crossed_fingers: this may just work. 
The following 'create_release' step is probably not needed, but you may know better.